test/functional/tests/incremental_load/test_incremental_load.py

```diff
@@ -0,0 +1,664 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+from datetime import timedelta
+from random import shuffle
+
+import pytest
+import time
+from api.cas import casadm, init_config, cli, cli_messages
+from api.cas.cache_config import CacheStatus, SeqCutOffPolicy
+from api.cas.core import CoreStatus, CacheMode, CleaningPolicy, FlushParametersAlru, File
+from api.cas.statistics import CacheStats
+from core.test_run import TestRun
+from storage_devices.disk import DiskTypeSet, DiskType, DiskTypeLowerThan
+from test_tools.dd import Dd
+from test_tools.disk_utils import Filesystem
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import IoEngine, ReadWrite
+from test_utils.output import CmdException
+from test_utils.size import Size, Unit
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_attach_core_to_incomplete_cache_volume():
+    """
+        title: Test for attaching device to inactive cache volume.
+        description: |
+          Try to attach core device to inactive cache volume and check if it is visible in OS
+          properly.
+        pass_criteria:
+          - No kernel error
+          - Core status change successfully
+          - Cache loads with inactive core device
+    """
+    with TestRun.step("Prepare devices."):
+        devices = prepare_devices([("cache", 1), ("core", 1)])
+        cache_dev = devices["cache"].partitions[0]
+        core_dev = devices["core"].partitions[0]
+        plug_device = devices["core"]
+    with TestRun.step("Start cache and add core."):
+        cache = casadm.start_cache(cache_dev, force=True)
+        core = cache.add_core(core_dev)
+    with TestRun.step("Create init config file using current CAS configuration."):
+        init_config.create_init_config_from_running_configuration()
+    with TestRun.step("Stop cache."):
+        cache.stop()
+    with TestRun.step("Load cache."):
+        casadm.load_cache(cache_dev)
+    with TestRun.step("Check if there is CAS device in /dev and core is in active status."):
+        core.check_if_is_present_in_os()
+        if core.get_status() != CoreStatus.active:
+            TestRun.fail("Core should be in active state.")
```

Comment:
general - I think that current state could be logged as well

Reply:
Done